spring-boot-project/spring-boot-actuator-autoconfigure/src/docs/asciidoc/endpoints/startup.adoc

```diff
@@ -16,7 +16,9 @@ The resulting response is similar to the following:
 
 include::{snippets}/startup/http-response.adoc[]
 
-
+NOTE: The above call resets the application startup steps buffer - subsequent calls to the endpoint will
```

Comment:
This may need some wordsmith polishing - it feels like I'm bleeding impl details to the doc.